tsbridge/metric.go

```diff
@@ -38,6 +40,20 @@ type Metric struct {
 	Record    storage.MetricRecord
 }
 
+// UpdateMetrics contains all external metric dependencies
+type UpdateMetrics struct {
+	SDClient       StackdriverAdapter
+	StatsCollector *StatsCollector
+}
+
+// NewExternalMetrics returns a UpdateMetrics struct.
+func NewUpdateMetrics(ctx context.Context, sd *stackdriver.Adapter, sc *StatsCollector) *UpdateMetrics {
```

Comment:
readability nit:

For a func that just returns a struct with passed in values, `func New(...)` is the preferred convention. 

`NewUpdateMetrics.UpdateMetrics` is the pointer, which is redundant in naming.

`New.UpdateMetrics` is a lot cleaner.

Reply:
Changed to func `New(...)`.